src/codegen/utilities/interpreter_helpers.py

```diff
@@ -34,12 +34,24 @@
     "SetSyncPulseTimeWhen",
 ]
 
+LIBRARY_INTERPRETER_IGNORED_FUNCTIONS = [
```

Comment:
This is a temporary structure, used to handle these functions. This would be removed once logic has been added to handle these function calls